app/controllers/clients_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
   def new
     @client = Client.new
     @client.address = Address.new
```

Comment:
Rails way of doing `@client.address = Address.new` is just `@client.address.build`